scripts/docker-build.sh

```diff
@@ -80,7 +80,7 @@ build_timescaledb()
         cd /build/debug \
         && git config --global --add safe.directory /src \
         && cmake -DGENERATE_DOWNGRADE_SCRIPT=${GENERATE_DOWNGRADE_SCRIPT} -DENABLE_DEBUG_UTILS=off -DUSE_OPENSSL=${USE_OPENSSL} -DCMAKE_BUILD_TYPE=${BUILD_TYPE} /src \
-        && make && make install \
+        && make -j 4 && make install \
```

Comment:
Maybe even `-j $(nproc)`?

Reply:
I thought `nproc` is not available on Alpine Linux. But you are right, it's available. PR is changed. Thanks.